app/src/components/market/sections/market_profile/market_status/closed.tsx

```diff
@@ -377,7 +368,7 @@ const Wrapper = (props: Props) => {
     isScalar,
     balances.map(balance => balance.shares),
     payouts,
-    finalAnswerPercentage,
+    Number(finalAnswerPercentage),
```

Comment:
Looks like user winnings data is getting jumbled, I think because of this line
![image](https://user-images.githubusercontent.com/30579067/114220665-22198380-9921-11eb-847c-f3b3e52e7ae5.png)

Reply:
👀